types/tabulator-tables/index.d.ts

```diff
@@ -1003,7 +1003,7 @@ You can pass an optional additional property with sorter, sorterParams that shou
     type CustomAccessorParams = {} | ((value: any, data: any, type: "data" | "download" | "clipboard", column?: ColumnComponent) => any);
     type ColumnCalc = "avg" | "max" | "min" | "sum" | "concat" | "count" | ((values: any[], data: any[], calcParams: {}) => number);
     type ColumnCalcParams = (values: any, data: any) => any;
-    type Formatter = "plaintext" | "textarea" | "html" | "money" | "image" | "datetime" | "datetimediff" | "link" | "tickCross" | "color" | "star" | "traffic" | "progress" | "lookup" | "buttonTick" | "buttonCross" | "rownum" | "handle" | ((cell: CellComponent, formatterParams: {}, onRendered: EmptyCallback) => string | HTMLElement);
+    type Formatter = ((cell: CellComponent, formatterParams: {}, onRendered: EmptyCallback) => string) | string | "plaintext" | "textarea" | "html" | "money" | "image" | "datetime" | "datetimediff" | "link" | "tickCross" | "color" | "star" | "traffic" | "progress" | "lookup" | "buttonTick" | "buttonCross" | "rownum" | "handle" | ((cell: CellComponent, formatterParams: {}, onRendered: EmptyCallback) => string | HTMLElement);
```

Comment:
The custom formatter at the end handles the one you copied on the front. The custom formatter should return either a string or html element per the docs.

http://tabulator.info/docs/4.2/format#format-builtin

Also, I don't see where you can add a plain string as a Formatter option. This negates the type benefits as well. I would undo this one too.